### PR TITLE
fix: sip10 tokens ledger send form

### PR DESCRIPTION
**Pull request by alter-eggo**
> Try out this version of Leather — [download extension builds](https://github.com/leather-wallet/extension/actions/runs/6795504762).

Fix broken sip10 tokens send flow in ledger mode

### CODE REVIEW

File: src/app/features/asset-list/components/stacks-ledger-assets.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    <>
+      <CryptoCurrencyAssetItem
+        assetBalance={stxEffectiveBalance}
+        usdBalance={stxEffectiveUsdBalance}
+        icon={<StxAvatar />}
+        address={currentAccount?.address || ''}
+        rightElement={hasStacksKeys ? undefined : <ConnectLedgerAssetBtn chain="stacks" />}
+      />
+      {hasStacksKeys && <StacksFungibleTokenAssetList assetBalances={stacksFtAssetBalances} />}
+    </>
```
Comment (kyranjamie) [inline src/app/features/asset-list/components/stacks-ledger-assets.tsx]: Is `hasStacksKeys` needed? We pass the `assetBalances` to this component, and we could only ever know what they are if the user does have stacks keys.

**Review by kyranjamie — APPROVED** [review]
Thanks for fixing @alter-eggo. 

Separately, let's think of a solution for this that doesn't involve any `whenWallet` logic. Better than these components are rendered conditionally based on the existence of a key, rather than wallet type.

But, as this is a P1, let's merge and release this ASAP.

**Comment by kyranjamie** [comment]
Also please amend commit with ref to #4491

File: src/app/features/asset-list/components/stacks-ledger-assets.tsx (hunk 1 above)
Comment (alter-eggo) [reply src/app/features/asset-list/components/stacks-ledger-assets.tsx]: yeah, checked, there is no need in `hasStacksKeys` here

**Pull request merged**